configure.ac

```diff
@@ -1754,7 +1754,7 @@ if test "$bitcoin_enable_qt" != "no"; then
     AC_MSG_WARN([xgettext is required to update qt translations])
   fi
 
-  AC_MSG_CHECKING([whether to build test_bitcoin-qt])
+  AC_MSG_CHECKING([whether to build test_vertcoin-qt])
```

Comment:
Next time let's leave this and line 1768 as test_bitcoin as the binary names have not been branded

Reply:
I put these back to to the original values.